packages/edit-post/src/plugins/index.js

```diff
@@ -14,11 +14,13 @@ import ManageBlocksMenuItem from './manage-blocks-menu-item';
 import KeyboardShortcutsHelpMenuItem from './keyboard-shortcuts-help-menu-item';
 import ToolsMoreMenuGroup from '../components/header/tools-more-menu-group';
 import WelcomeGuideMenuItem from './welcome-guide-menu-item';
+import BlockPatterns from './block-patterns';
 
 registerPlugin( 'edit-post', {
 	render() {
 		return (
 			<>
+				<BlockPatterns />
```

Comment:
Cool idea to use Plugins API. Once we have the design sorted out, we can add a new slot in the inserter and everything should still work.